packages/foreman/rubygem-actioncable/rubygem-actioncable.spec

```diff
@@ -28,7 +28,7 @@ BuildArch: noarch
 Provides: %{?scl_prefix}rubygem(%{gem_name}) = %{version}
 # end specfile generated dependencies
 
-Obsoletes: tfm-ror52-rubygem-%{gem_name} <= 5.2.1
+Obsoletes: tfm-ror52-rubygem-%{gem_name} < %{version}-%{release}
```

Comment:
This essentially will, for all time, Obsolete any tfm-ror52 version-release for this package as this will change everytime the package get's release or version updated. If that's the case, what is the benefit then to including version-release specification?

Reply:
Technically, we could re-create "tfm-ror52-something" with a newer release.
Effectively: make rpmlint shut up, as I don't see us having a rails 5.2 scl ever again.

Either way, the old way was not working as it was not matching the last released ror52 version (as it was lacking the release bit).

Reply:
I think the old worked because it was `<=`. On my system most packages were removed automatically, save for those where I made a PR.

Reply:
So I retested this, and it's indeed sufficient. So while I don't fully understand it, I'll back out that change, as it's not really needed.